llama-index-integrations/embeddings/llama-index-embeddings-cohere/pyproject.toml

```diff
@@ -32,7 +32,6 @@ version = "0.1.5"
 [tool.poetry.dependencies]
 python = ">=3.8.1,<4.0"
 llama-index-core = "^0.10.1"
-cohere = "^5.1.1"
```

Comment:
feels weird to have to remove the dep on cohere lol

Reply:
But to work with cohere binary embeddings we need cohere >= 5.1.1 version.

Reply:
I know lol. This was more of a quick fix/workaround. We have done similar in other integrations 😞 packages.

BUT, if we want to resolve it better, then I think the issue lies here, where langchain extra sets a conflicting range for `cohere`? Could be wrong. Please revert if have a better solution! 🙏

https://github.com/run-llama/llama_index/blob/ff73754c5b68e9f4e49b1d55bc70e10d18462bce/llama-index-core/poetry.lock#L2662

Reply:
oh yeah. Its weird that langchain uses older package version.

Reply:
I think I fixed this by ignoring langchain deps in the BUILD file